Dockerfile

```diff
@@ -1,21 +1,21 @@
 FROM golang:alpine as builder
 
-ADD . /usr/src/rdma-cni
+COPY . /usr/src/rdma-cni
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 
-RUN apk add --update --virtual build-dependencies build-base && \
-    cd /usr/src/rdma-cni && \
+WORKDIR /usr/src/rdma-cni
+RUN apk add --no-cache --virtual build-dependencies build-base=~0.5 && \
```

Comment:
why  build-base=~0.5?

Reply:
essentially copied either from sriov-cni or sriov-device-plugin.
hadolint complains package version should be fixed.

seeing we only got 0.5 
https://pkgs.alpinelinux.org/packages?name=build-base&branch=edge&repo=&arch=&maintainer=

Reply:
ok